src/requests/misc.jl

```diff
@@ -9,7 +9,7 @@ function process(r::JSONRPC.Request{Val{Symbol("julia/lint-package")},Nothing},
 
 JSONRPC.parse_params(::Type{Val{Symbol("julia/toggle-lint")}}, params) = TextDocumentIdentifier(params["textDocument"])
 function process(r::JSONRPC.Request{Val{Symbol("julia/toggle-lint")},TextDocumentIdentifier}, server)
-    doc = server.documents[URI2(r.uri)]
+    doc = getdocument(server, URI2(r.uri))
     doc._runlinter = !doc._runlinter
 end
```

Comment:
Note: Should remove from here and julia-vscode.

Reply:
https://github.com/julia-vscode/LanguageServer.jl/pull/541. As far as I can tell there was no corresponding code in julia-vscode anymore, we must have removed it there already.